src/components/MyProfile.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  myReservedRockets,
+  getReservedRockets,
+  getRocketsStatus,
+} from '../redux/rocketsSlice';
+import { getReservedMissionTitles } from '../redux/missionSlice';
+import '../styles/MyProfile.css';
+import { getReservedDragons } from '../redux/dragonSlice';
+
+const MyProfile = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector(getReservedRockets);
+  const rocketStatus = useSelector(getRocketsStatus);
+  const reservedMissionTitles = useSelector(getReservedMissionTitles);
+  const dragons = useSelector(getReservedDragons);
+
+  useEffect(() => {
+    dispatch(myReservedRockets());
+  }, [rocketStatus, dispatch]);
+
+  let renderReserved = '';
+  if (rockets.length) {
+    renderReserved = rockets.map((data) => (
+      <li className="filter-list" key={data.id}>
+        <span>{data.rocket_name}</span>
+      </li>
+    ));
+  }
+
+  let renderedDragons = '';
+  if (dragons.length) {
+    renderedDragons = dragons.map((data) => (
+      <li className="filter-list" key={data.id}>
+        <span>{data.name}</span>
+      </li>
+    ));
+  }
+
+  let renderReservedMissions = '';
+  if (reservedMissionTitles.length) {
+    renderReservedMissions = reservedMissionTitles.map((title) => (
+      <li className="filter-list" key={title}>
+        <span>{title}</span>
+      </li>
+    ));
+  }
```

Comment:
- [ ] What a great of `ES6 map()` here and it really worked just fine except that the requirement asked that you opt for `filter()` instead.

![image](https://github.com/the-faizmohammad/spacex-travellers-hub/assets/99715640/31668aa9-c2d0-4b37-80e6-af91a888246c)
- To match up with the above, could you use `filter()` for your filtering?